RefugeeLand.Core.Api/Brokers/Storages/StorageBroker.SkillSets.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO DELIVER HUMANITARIAN AID, HOPE AND LOVE
+// -------------------------------------------------------
+
+using Microsoft.EntityFrameworkCore;
+using RefugeeLand.Core.Api.Models.SkillSets;
+
+namespace RefugeeLand.Core.Api.Brokers.Storages
+{
+    public partial class StorageBroker
+    {
```

Comment:
just a thought...would it be not be useful to split it out into an interface so that it can be used across multiple apps? for example a local db on the mobile app and cloud db for the web API?

Reply:
@bramburn 
I'm not sure to understand exactly what you are saying. So let's have a chat over a video call, this way I'll be able to fully understand what you are proposing. Sounds good?